mitodl/mitxonline
# Updates to ancillary pages

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/3413

### Description (What does it do?)

A bunch of pages in MITx Online needed some updates to be in-line with the redesign work for the course pages, catalog, and front page. This adds those based on the styling provided by Bilal in https://www.figma.com/file/Gs4zIhOFv5gVvafawwl6PF/MITx-Online?node-id=4450%3A6795&mode=dev .

### Screenshots (if appropriate):

Sign in/create account:

![Screenshot 2024-03-04 at 2 06 59 PM](https://github.com/mitodl/mitxonline/assets/945611/739a308c-50e8-45c4-b40a-25df9b8cb3df)
![Screenshot 2024-03-04 at 2 07 10 PM](https://github.com/mitodl/mitxonline/assets/945611/49fa66bd-b8d4-4cf4-860c-d7a58cce9e76)
![Screenshot 2024-03-04 at 2 10 00 PM](https://github.com/mitodl/mitxonline/assets/945611/3ec1e907-9fd7-4b75-bdf4-2a53e36de062)
![Screenshot 2024-03-04 at 2 10 23 PM](https://github.com/mitodl/mitxonline/assets/945611/f87c682e-b0e0-44ed-8a23-749321b47382)
![Screenshot 2024-03-04 at 2 10 50 PM](https://github.com/mitodl/mitxonline/assets/945611/6f70e1d4-eb57-411c-bf04-1be5c47d62c9)

Reset password:
![image](https://github.com/mitodl/mitxonline/assets/945611/bd220dfe-eabe-45cc-8c34-e0a684fcce1c)
![image](https://github.com/mitodl/mitxonline/assets/945611/fd565ca6-3f23-457e-b980-f1414e2a08c8)
![image](https://github.com/mitodl/mitxonline/assets/945611/c947a899-7f49-450c-8d02-052e7b45ca83)



Dashboard:
![Screenshot 2024-03-04 at 2 11 25 PM](https://github.com/mitodl/mitxonline/assets/945611/377e89fa-1ce9-48bd-9b57-25f4abce36e6)
![Screenshot 2024-03-04 at 2 05 59 PM](https://github.com/mitodl/mitxonline/assets/945611/f666f52a-426c-4504-9ea6-c5b124128cb1)

Order history:
![Screenshot 2024-03-04 at 2 06 48 PM](https://github.com/mitodl/mitxonline/assets/945611/5a5ec677-875c-4b76-bb03-6a186f37a6f7)

Financial assistance:
![Screenshot 2024-03-04 at 2 06 12 PM](https://github.com/mitodl/mitxonline/assets/945611/d307561f-386e-476c-bdf9-bade978f0389)
![Screenshot 2024-03-04 at 2 06 24 PM](https://github.com/mitodl/mitxonline/assets/945611/8ade5d6c-2891-403e-8a93-c132a1a94724)

Profile:
![image](https://github.com/mitodl/mitxonline/assets/945611/f5a5a12c-cb4a-48f7-8a89-6445b1d5287e)

![Screenshot 2024-03-04 at 2 05 47 PM](https://github.com/mitodl/mitxonline/assets/945611/7eab343d-70a5-43d5-961a-bf84c23ad448)

Program record:
![Screenshot 2024-03-04 at 2 05 25 PM](https://github.com/mitodl/mitxonline/assets/945611/c622c1f4-3307-4956-a3e8-cac421eac0a9)

Cart:
![image](https://github.com/mitodl/mitxonline/assets/945611/63d4bb38-b95d-4a95-8762-c60913a1a63c)


### How can this be tested?

Check the pages against the design.

Also, note:
* The blue page header banner should appear when the learner is logged in. (So, on the login/forgot password workflows, there should _not_ be a banner.) 
   * I'm not sure how this applies to the learner record.
* Some of the figures will be compensated a bit to accomodate the fact that Figma doesn't appear to accommodate descenders in margins and padding. (i.e. if the design lists 20px bottom margin, that 20px also includes the tails on "g j y", etc. but the browser does _not_ - the descenders are above the margin.) 

There are no functionality updates here - this should all be design work.
